app/src/css/_base.scss

```diff
@@ -42,6 +42,15 @@ body {
   height: 100%;
 }
 
+row {
+    flexDirection: 'row';
+    height: 50;
+    justifyContent: 'center';
+    alignItems: 'center';
+    position: 'absolute'; //Here is the trick
+    bottom: 0;
+}
```

Comment:
I don't think this is doing anything, since `row` is not a valid selector and some of the properties and invalid